services/QuillLMS/client/app/bundles/Diagnostic/components/questions/responseComponent.jsx

```diff
@@ -80,6 +82,12 @@ class ResponseComponent extends React.Component {
     this.clearResponses();
   }
 
+  updateResponse = (id, response) => {
+    const newResponses = this.state.responses
```

Comment:
We extract `{ responses, } = this.state` on 236, so we might as well do so here and below on line 165.